starboard/build/config/starboard_target_type.gni

```diff
@@ -37,6 +41,17 @@ template("starboard_platform_target") {
     }
     if (target_type == "shared_library") {
       build_loader = false
+      install_target = false
+      if (!is_host_win) {
+        output_extension = starboard_shared_library_extension
+        if (use_contrib_cast) {
+          output_name = cast_library_name
+        } else {
+          output_name = "starboard"
```

Comment:
I think ideally we would rename the target to "starboard" instead of just changing the `output_name`—changing the `output_name`, especially to something so different, could cause conflicts down the line (i.e. what if someone tried changing the type of the existing "starboard" target?)

Reply:
ok that's a much bigger change then where all places which rely on //starboard get renamed to //starboard_group maybe ?

Also note that for windows you would have to change the output name if you want to include the api version in parallel with linux

Reply:
Yep starboard_group is fine, starboard_module, starboard_shim, whatever sounds best to you

For windows what name are we trying to create? `starboard.lib` would be the standard name, and I assume we'd do `starboard.lib.16` (for example) to match linux, but it seems like here you're doing `starboard.16.lib`? Whatever the case, I believe we can just rely on `output_extension` exclusively and never change `output_name`

Reply:
For windows the standard acc to the microsoft link would be libstarboard.16.dll  - i dont know how much we care about the windows name - ill ask on the group and if no  one has any strong views we can keep it libstarboard.dll . For windows you need the .lib as the suffix.   
[earn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-assemblies-and-dlls](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-assemblies-and-dlls)

Reply:
Got it—regardless you could do 16.lib (or 16.dll? not sure which it is) as the output extension without changing the output name

Reply:
im going to keep this as libstarboard.dll -> because different windows platforms ps4/xb1/nxswitch all have different extensions(dll, .elf,, .nsp) - so that would be harder to implement for non-linux platforms.